docs/getting_started.rst

```diff
@@ -20,32 +19,46 @@ LXDock should build very easily on Linux provided you have LXD available on your
 Prerequisite: install LXD
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You may want to skip this section if you already have a working installation of LXD on your system.
+You may want to skip this section if you already have a working installation
+of LXD on your system.
 
-For Debian and Ubuntu, the following command will ensure that LXD is installed:
+For Debian or Ubuntu 16.04 and higher if prefer to stick with apt:
```

Comment:
The reason I put 16.04+ here for the apt version, is because 14.04 no longer has access to the PPA so will have to use the Snap on 14.04 now